mobile/src/features/Transactions/useCases/TxHistory/ActionsBanner.tsx

```diff
@@ -66,7 +57,8 @@ export const ActionsBanner = (props: {disabled: boolean}) => {
       slippage_tolerance: 1,
```

Comment:
is passing 1 here correct? looks like a bug

Reply:
This is for tracking swap initiated, we don't know the actual slippage here, and 1 is the default value.